.github/workflows/e2e-compatibility.yaml

```diff
@@ -208,7 +235,16 @@ jobs:
     with:
       test-file-directory: "${{ needs.determine-test-directory.outputs.test-directory }}"
       test-suite: "ica-gov-chain-a"
-  
+
+  ica-gov-chain-b:
```

Comment:
I don't think we have an unreleased for the groups test suite yet?

Reply:
This is added so that we can run the tests where chain a is fixed and chain b is one of the releases that support the feature. Basically the mirror of `ica-gov-chain-a`.

Reply:
The gov/groups test will eventually be needed for the unreleased section? ie release/v7.0.x vs release/7.1.x?

Reply:
True, I will add those and also localhost and transfer-authz.